Documentation/platforms/container-linux-systemd.md

```diff
@@ -0,0 +1,166 @@
+# Run etcd on Container Linux with systemd
+
+The following guide shows how to run etcd with [systemd][systemd-docs] under [Container Linux][container-linux-docs].
+
+## Provisioning an etcd cluster
+
+Cluster bootstrapping in Container Linux is simplest with [Ignition][container-linux-ignition]; `coreos-metadata.service` dynamically fetches the machine's IP for discovery. Note that etcd's discovery service protocol is only meant for bootstrapping, and cannot be used with runtime reconfiguration or cluster monitoring.
+
+The [Container Linux Config Transpiler][container-linux-ct] compiles etcd configuration files into Ignition configuration files:
+
+```bash
```

Comment:
If you put the contents of Container Linux Configs into a code fence with the info string `yaml container-linux-config`, it will render nicely on GitHub and coreos.com. Our site will further annotate the block with the Ignition config, so the second part of your block won't be necessary.